llvm/unittests/IR/BasicBlockDbgInfoTest.cpp

```diff
@@ -1525,4 +1525,58 @@ TEST(BasicBlockDbgInfoTest, DbgMoveToEnd) {
   EXPECT_FALSE(Ret->hasDbgRecords());
 }
 
+TEST(BasicBlockDbgInfoTest, DbgKnownSentinelCrash) {
+  return;
```

Comment:
Should the early return here be removed?

Reply:
Oops, yep, will do